package.json

```diff
@@ -657,6 +703,10 @@
         {
           "command": "azure-iot-toolkit.startMonitorIoTHubMessageWithAbbreviation",
           "when": "never"
+        },
+        {
+          "command": "azure-iot-dps.loadMore",
+          "when": "never"
```

Comment:
shall we also disable refresh in command palette?

Reply:
The refresh command in command palette can be used to refresh the entire treeview. IoT Hub also provides refresh command in command palette to refresh device treeview.
So we can keep the refresh command in command palette to provide additional way to refresh?

Reply:
I am asking this because I found refresh method needs "Node", if "Node" is undefined, it could refresh all, Then its fine to have it in command palette